lsusb.py: fix parse_usb_ids

cmheia opened the pull request:
The commit 10144ac broke the usb.ids parser by renaming 'id' to 'vid' instead of 'cid' - fix that.

gregkh reviewed (changes requested):
Need a full name, not an email alias, on a signed-off-by line please.

cmheia committed:
lsusb.py: fix parse_usb_ids

The commit 10144ac broke the usb.ids parser by renaming 'id' to 'vid' instead of 'cid' - fix that.

Signed-off-by: MingHao Chen <cmheia@gmail.com>

cmheia commented:
> Need a full name, not an email alias, on a signed-off-by line please.

done.
